### PR TITLE
[CDAP-6690] Pretty-print get dataset properties output in CLI

### PR DESCRIPTION
**Pull request by anew**
Output now looks like this:
```
cdap (http://MacBook-Pro-2.local:11015/namespace:default)> get dataset instance properties mytable
+=========================================+
| property                       | value  |
+=========================================+
| table.colum.family             | xx     |
| table.conflict.detection.level | COLUMN |
| table.ttl                      | 100    |
+=========================================+
```

### CODE REVIEW

File: cdap-cli/src/main/java/co/cask/cdap/cli/command/GetDatasetInstancePropertiesCommand.java [hunk 1]
```diff
@@ -51,7 +57,17 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
       arguments.get(ArgumentName.DATASET.toString()));
 
     Map<String, String> properties = datasetClient.getProperties(instance);
-    output.printf(GSON.toJson(properties));
```
Comment (sagarkapare) [inline cdap-cli/src/main/java/co/cask/cdap/cli/command/GetDatasetInstancePropertiesCommand.java:54]: unused `GSON`?

**Comment by sagarkapare** [comment]
just one comment about unused variable, rest LGTM 👍

**Comment by anew** [comment]
Thanks, I will address and squash

File: cdap-cli/src/main/java/co/cask/cdap/cli/command/GetDatasetInstancePropertiesCommand.java (hunk 1 above)
Comment (anew) [reply cdap-cli/src/main/java/co/cask/cdap/cli/command/GetDatasetInstancePropertiesCommand.java:54]: Oh I missed that it wasn't used anywhere else. Good catch.

File: cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java [hunk 2]
```diff
@@ -348,7 +348,7 @@ public void testDataset() throws Exception {
     testCommandOutputContains(cli, "create dataset instance " + datasetType.getName() + " " + datasetName + " \"a=1\"",
                               "Successfully created dataset");
     testCommandOutputContains(cli, "list dataset instances", FakeDataset.class.getSimpleName());
-    testCommandOutputContains(cli, "get dataset instance properties " + datasetName, "\"a\":\"1\"");
+    testCommandOutputContains(cli, "get dataset instance properties " + datasetName, "a,1");
```
Comment (anew) [inline cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java:352]: changing the output from JSON to tabular form (and in unit tests, the table formatter uses ",") broke this test.

**Review by sagarkapare — COMMENTED** [review]
LGTM 👍

**Comment by anew** [comment]
build passed: http://builds.cask.co/browse/CDAP-RUT545-2

squash & merge

**Pull request merged**